modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/seedlist/types.scala

```diff
@@ -1,20 +1,49 @@
 package org.tessellation.sdk.infrastructure.seedlist
 
+import cats.Order
+import cats.syntax.contravariant._
+import cats.syntax.option._
+
 import org.tessellation.schema.peer.PeerId
-import org.tessellation.security.hex.Hex
+import org.tessellation.schema.trust._
 
-import fs2.data.csv.RowDecoder
+import fs2.data.csv._
 import fs2.data.csv.generic.semiauto.deriveRowDecoder
 
 object types {
 
-  case class SeedlistCSVEntry(id: String) {
+  case class SeedlistEntry(
+    peerId: PeerId,
+    ipAddress: Option[String],
+    alias: Option[String],
```

Comment:
it would be best to create a newtype for alias

Reply:
`alias` is now the `Alias` type:

```
type Alias = String
```

Reply:
`type Alias = String` is not a newtype and does not prevent human mistakes (you can assign any string there and it won't throw a compile error). please take a look how `PeerId` is created, you already used that construction.